CODEOWNERS

```
@@ -121,6 +121,9 @@
 /task/oci-copy          @ralphbean
 /task/oci-copy-oci-ta   @ralphbean
 
+# renovate groupName=modelcar-oci
+/task/modelcar-oci-ta   @Victoremepunto
```

I can help co-maintain this.

```suggestion
/task/modelcar-oci-ta   @Victoremepunto @ralphbean 
```